src/heretic/mn_menu.c

```diff
@@ -1125,7 +1125,7 @@ static void DrawOptionsMenu(void)
 
 static void DrawOptions2Menu(void)
 {
-    DrawSlider(&Options2Menu, 1, 9, screenblocks - 3);
+    DrawSlider(&Options2Menu, 1, 9, (screenblocks < 12 ? screenblocks : 11) - 3);
```

Comment:
I think we have a `BETWEEN()` macro for cases like this.

Reply:
The macro is now used in all three occasions.